<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-QU6JJGH" failures="5" tests="32" name="Surefire test" time="54.248" errors="0" timestamp="2024-03-05T19:13:08 IST">
  <testcase classname="com.example.testclasses.BaseClass" name="testUserLogin" time="1.233"/>
  <testcase classname="com.example.testclasses.UserClass" name="deleteUserByIdWithInvalidId" time="0.844"/>
  <testcase classname="com.example.testclasses.UserClass" name="deleteUserByIdWithInvalidIdAndToken" time="0.846"/>
  <testcase classname="com.example.testclasses.UserClass" name="deleteUserByIdWithInvalidToken" time="3.091"/>
  <testcase classname="com.example.testclasses.UserClass" name="deleteUserByIdWithValidDetails" time="2.998">
    <failure type="java.lang.AssertionError" message="Status code not 200
Expected: &amp;lt;200&amp;gt;
     but: was &amp;lt;404&amp;gt;">
      <![CDATA[java.lang.AssertionError: Status code not 200
Expected: <200>
     but: was <404>
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at com.example.testclasses.UserClass.deleteUserByIdWithValidDetails(UserClass.java:546)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- deleteUserByIdWithValidDetails -->
  <testcase classname="com.example.testclasses.UserClass" name="getAllUsers" time="1.227"/>
  <testcase classname="com.example.testclasses.UserClass" name="getAllUsersWithInvalidToken" time="0.824"/>
  <testcase classname="com.example.testclasses.UserClass" name="getCatchesByInvalidId" time="0.836"/>
  <testcase classname="com.example.testclasses.UserClass" name="getCatchesByInvalidToken" time="0.83"/>
  <testcase classname="com.example.testclasses.UserClass" name="getCatchesByInvalidTokenAndInvalidId" time="0.84"/>
  <testcase classname="com.example.testclasses.UserClass" name="getCatchesByUserId" time="0.897"/>
  <testcase classname="com.example.testclasses.UserClass" name="getFriendsByInvalidId" time="0.844"/>
  <testcase classname="com.example.testclasses.UserClass" name="getFriendsByInvalidToken" time="0.818"/>
  <testcase classname="com.example.testclasses.UserClass" name="getFriendsByInvalidTokenAndInvalidId" time="0.802"/>
  <testcase classname="com.example.testclasses.UserClass" name="getFriendsByUserId" time="0.865"/>
  <testcase classname="com.example.testclasses.UserClass" name="getReferenceObjById" time="4.912">
    <failure type="org.json.JSONException" message="JSONObject[&amp;quot;data&amp;quot;] not found.">
      <![CDATA[org.json.JSONException: JSONObject["data"] not found.
at org.json.JSONObject.get(JSONObject.java:595)
at org.json.JSONObject.getJSONObject(JSONObject.java:805)
at com.example.testclasses.UserClass.getReferenceObjById(UserClass.java:622)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- getReferenceObjById -->
  <testcase classname="com.example.testclasses.UserClass" name="getReferenceObjByInvalidId" time="0.857"/>
  <testcase classname="com.example.testclasses.UserClass" name="getReferenceObjByInvalidToken" time="4.667"/>
  <testcase classname="com.example.testclasses.UserClass" name="getReferenceObjByInvalidTokenAndId" time="0.79"/>
  <testcase classname="com.example.testclasses.UserClass" name="getUserById" time="1.001"/>
  <testcase classname="com.example.testclasses.UserClass" name="getUserByInvalidId" time="0.806"/>
  <testcase classname="com.example.testclasses.UserClass" name="getUserByInvalidToken" time="0.833"/>
  <testcase classname="com.example.testclasses.UserClass" name="getUserByInvalidTokenAndInvalidId" time="0.784"/>
  <testcase classname="com.example.testclasses.UserClass" name="updateUserByUserId" time="1.826">
    <failure type="java.lang.AssertionError" message="
Expected: &amp;quot;TestUser.exactink@gmail.com&amp;quot;
     but: was &amp;quot;testuser.exactink@gmail.com&amp;quot;">
      <![CDATA[java.lang.AssertionError: 
Expected: "TestUser.exactink@gmail.com"
     but: was "testuser.exactink@gmail.com"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:6)
at com.example.testclasses.UserClass.updateUserByUserId(UserClass.java:275)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- updateUserByUserId -->
  <testcase classname="com.example.testclasses.UserClass" name="updateUserByUserIdWithInvalidEmail" time="1.929">
    <failure type="java.lang.AssertionError" message="
Expected: &amp;quot;TestUser.exactink@Example.com&amp;quot;
     but: was &amp;quot;testuser.exactink@example.com&amp;quot;">
      <![CDATA[java.lang.AssertionError: 
Expected: "TestUser.exactink@Example.com"
     but: was "testuser.exactink@example.com"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:6)
at com.example.testclasses.UserClass.updateUserByUserIdWithInvalidEmail(UserClass.java:339)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- updateUserByUserIdWithInvalidEmail -->
  <testcase classname="com.example.testclasses.UserClass" name="updateUserByUserIdWithInvalidToken" time="0.774"/>
  <testcase classname="com.example.testclasses.UserClass" name="updateUserByUserIdWithInvalidTokenAndInvalidId" time="0.795"/>
  <testcase classname="com.example.testclasses.UserClass" name="updateUserByUserIdWithInvalidUserId" time="0.812"/>
  <testcase classname="com.example.testclasses.UserClass" name="updateUserWithEmojiInFirstAndLastName" time="1.858"/>
  <testcase classname="com.example.testclasses.UserClass" name="updateUserWithOnlyRequiredField" time="2.027"/>
  <testcase classname="com.example.testclasses.UserClass" name="uploadReference_Obj" time="3.845">
    <failure type="org.json.JSONException" message="JSONObject[&amp;quot;data&amp;quot;] not found.">
      <![CDATA[org.json.JSONException: JSONObject["data"] not found.
at org.json.JSONObject.get(JSONObject.java:595)
at org.json.JSONObject.getJSONObject(JSONObject.java:805)
at com.example.testclasses.UserClass.uploadReference_Obj(UserClass.java:583)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- uploadReference_Obj -->
  <testcase classname="com.example.testclasses.UserClass" name="uploadReference_ObjWithInvalidToken" time="3.214"/>
</testsuite> <!-- Surefire test -->
